### PR TITLE
Add skcms module

### PR DESCRIPTION
**Pull request by eustas**
@bazel-io skip_check unstable_url

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (skcms) have been updated in this PR. Please review the changes.

**Comment by eustas** [comment]
Pipelines are green now.

**Review by meteorcloudy — DISMISSED** [review]

File: modules/skcms/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://skia.org/",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```
Comment (meteorcloudy) [inline modules/skcms/metadata.json:5]: nit: Do you want to be list as maintainer for this module?

**Review by meteorcloudy — APPROVED** [review]

File: modules/skcms/metadata.json (hunk 1 above)
Comment (meteorcloudy) [reply modules/skcms/metadata.json]: I think it's fine.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**